mautic/mautic
# Fix characters in segment name and filters

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Fix for https://github.com/mautic/mautic/issues/3488
Characters like  `<` `>`  make validation issue during save segment If you use it in public name or in options of multiselect field. This PR fixed it




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create multiselect custom field and add option `< 3 years`
3. Then try create segment with public name `< 3 years` and select in filter created multiselect field value in `< 3 years`
4. Try save. It return value is not valid
5. After PR It should save properly
